client/src/login-component/index.js

```diff
@@ -1,87 +1,67 @@
-import React, { useState } from "react";
-// styling
-import "bootstrap/dist/css/bootstrap.min.css";
-
+import React, { useState } from 'react';
 // Bootstrap compponents
-import Button from "react-bootstrap/Button";
-import Form from "react-bootstrap/Form";
-import Container from "react-bootstrap/Container";
-import Row from "react-bootstrap/Row";
-import Col from "react-bootstrap/Col";
-import Signup from "../signup-component/index";
-
-function Login() {
-  const [username, setUsername] = useState("");
-  const [password, setPassword] = useState("");
-
-  // const handleSubmit = (e) => {
-  //     e.preventDefault();
-  //     // validate username and password here
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
 
-  //     // load post component
-  //     return (
-  //         <Post>
-  //         </Post>
-  //     );
-
-  //   };
+const Login = () => {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
 
   const handleSubmit = () => {
     // TODO: handle transition from login to post screen
```

Comment:
As part of the routing enhancement, could you also had a transition here to feed page? I used `Links` in an earlier provision and worked smoothly

Reply:
or how you did it in signup

Reply:
Yeah I'll use links.